examples/cordova-sample/MyApp/bin/sk_config_setup.rb

```diff
@@ -6,54 +6,103 @@
 
 require 'xcodeproj'
 
-workspace_path = 'platforms/ios/HelloCordova.xcworkspace'
-puts("Workspace: #{workspace_path}\n")
+def save_project_if_needed(project, modified = false)
+    unless modified
```

Comment:
I believe you can simplify this implementation with

```ruby
return unless modified
```

Reply:
Omg 🤦‍♂️

Reply:
✅